python/mxnet/kvstore.py

```diff
@@ -349,6 +349,101 @@ def row_sparse_pull(self, key, out=None, priority=0, row_ids=None):
             check_call(_LIB.MXKVStorePullRowSparse(
                 self.handle, mx_uint(len(ckeys)), ckeys, cvals, crow_ids, ctypes.c_int(priority)))
 
+    def set_compress(self, compress_params=None):
```

Comment:
set_gradient_compression(self, method, **kwargs)
same for c++ API

Reply:
Cool, I can replace the usage of a dictionary with kwargs.